src/Ledger/Epoch.lagda

```diff
@@ -156,9 +146,83 @@ toRwdAddr x = record { net = NetworkId ; stake = x }
+Relevant quantities are:
+\begin{itemize}
+  \item \AgdaArgument{prevPp}: Previous protocol parameters,
+    which correspond to the parameters during the epoch for which we are creating rewards.
+  \item \AgdaFunction{Δr₁}: Ada taken out of the reserves for paying rewards,
+    as determined by the \AgdaField{monetaryExpansion} protocol parameter.
+  \item \AgdaFunction{rewardPot}: Total amount of coin available for rewards this epoch, as
+described in section 6.4 of \textcite{shelley-delegation-design}.
+  \item \AgdaFunction{feeSS}: The fee pot, containing the transaction fees from the epoch.
+    We use the fee pot from the snapshot taken at the epoch boundary.
+    Note that fees are not explicitly removed from any account:
+    the fees come from transactions paying them and are accounted for whenever transactions are processed.
+  \item \AgdaFunction{Δt₁}: The proportion of the reward pot that will move to the treasury,
+    as determined by the \AgdaField{treasuryCut} protocol parameter.
+    The remaining pot is called the \AgdaFunction{R},
+    just as in section 6.5 of \textcite{shelley-delegation-design}.
+  \item \AgdaFunction{pstakego}:
+    Stake distribution used for calculating the rewards.
+    This is the oldest stake distribution snapshot, labeled ``go''.
+  \item \AgdaFunction{rs}: The calculated rewards.
+    As given by \AgdaFunction{maxPool}, each pool can receive a maximal amount,
+    determined by its performance.
+  \item \AgdaFunction{Δr₂}: The difference between the maximal amount of rewards
+    that could have been paid out if pools had been optimal,
+    and the actual rewards paid out.
+    This difference is returned to the reserves.
+  \item \AgdaFunction{÷₀}: Division operator that returns zero when the denominator is zero.
+\end{itemize}
+
+\begin{figure*}[h]
+\begin{code}
+ActiveSlotCoeff : ℚ
+ActiveSlotCoeff = (1 / 20)
+\end{code}
+\begin{code}[hide]
+  {{nonZero}}
+\end{code}
+\begin{code}
+createRUpd : PParams → ℕ → BlocksMade → EpochState → Coin → RewardUpdate
+createRUpd prevPp slotsPerEpoch b es total
+  = ⟦ Δt₁ , 0 - Δr₁ + Δr₂ , 0 - feeSS , rs ⟧
+  where
+    reserves = es .EpochState.acnt .Acnt.reserves
+    pstakego  = es .EpochState.ss .Snapshots.go
+    feeSS    = es .EpochState.ss .Snapshots.feeSS
+    stake = pstakego .Snapshot.stake
+    delegs  = pstakego .Snapshot.delegations
+    poolParams = pstakego .Snapshot.poolParameters
+
+    blocksMade = ∑[ m ← b ] m
+
+    rho = fromUnitInterval (prevPp .PParams.monetaryExpansion)
+    η = fromℕ blocksMade ÷₀ (fromℕ slotsPerEpoch * ActiveSlotCoeff)
+    Δr₁ = floor (ℚ.min 1 η * rho * fromℕ reserves)
+
+    rewardPot = pos feeSS + Δr₁
+    tau = fromUnitInterval (prevPp .PParams.treasuryCut)
+    Δt₁ = floor (tau * fromℤ rewardPot)
+    R = posPart (rewardPot - Δt₁)
+    circulation = total - reserves
+
+    rs = reward prevPp b R poolParams stake delegs circulation
+    Δr₂ = R - ∑[ c ← rs ] c
+\end{code}
+\caption{RewardUpdate Creation}
```

Comment:
```suggestion
\end{code}
\end{AgdaMultiCode}
\caption{RewardUpdate Creation}
```
So that there is one blank like between `ActiveSlotCoeff` and `createRUpd` instead of 2